src/software/new_geom/line_test.cpp

```diff
@@ -0,0 +1,16 @@
+#include "software/new_geom/line.h"
+
+#include <gtest/gtest.h>
+
+TEST(LineTest, two_points_constructor)
+{
+    Line l(Point(-1.0, 2.0), Point(3.0, 4.0));
+    EXPECT_EQ(l.toNormalUnitVector(), Vector(-1, 2).normalize());
+}
+
+TEST(LineTest, swapXY)
```

Comment:
Can we add a test for getting the normal vector?

Reply:
I'm not sure what that would test beyond what's being tested by `two_points_constructor` test; is there a particular type of line that you would like to see tested?

Reply:
Ah I didn't realize the `two_points_constructor` test used it. I think it would still be good to have a quick sanity check that the returned normal unit vector is what's expected

Reply:
Doesn't have to be anything crazy complicated

Reply:
Added a test where two different `Line`s' normal unit vectors are compared. Also added a test for a vertical `Line`.